src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,306 @@
+                    if (!String.IsNullOrEmpty(UpdateAsyncArgumentsParse(repositoryBranchUpdateHistory.Arguments)))
+                    {
+                        description.Append(UpdateAsyncArgumentsParse(repositoryBranchUpdateHistory.Arguments));
+                        break;
+                    }
+                    return;
+                // since this is not really an error, skipping this method
+                case "SynchronizePullRequestAsync":
+                    return;
+                // for all the other methods
+                default:
+                    description.Append("Repository :" + repositoryBranchUpdateHistory.Repository);
+                    break;
+            }
+            description.Append(Environment.NewLine);
+            description.Append("Branch Name : " + repositoryBranchUpdateHistory.Branch);
+            description.Append(Environment.NewLine);
+            description.Append("Error Message : " + repositoryBranchUpdateHistory.ErrorMessage);
+            description.Append(Environment.NewLine);
+            description.Append("Method : " + repositoryBranchUpdateHistory.Method);
+            description.Append(Environment.NewLine);
+            description.Append("Action : " + repositoryBranchUpdateHistory.Action);
+            description.Append(Environment.NewLine);
+            description.Append("/FYI : " + fyiHandles);
+            description.Append(Environment.NewLine + Environment.NewLine + Environment.NewLine);
+
+            // Get repo info used for create/ update gitHub issue
+            Octokit.Repository repo = await client.Repository.Get(_options.Owner, _options.Repository);
+            try
+            {
+                using (ITransaction tx = _stateManager.CreateTransaction())
+                {
+                    if (await gitHubIssueEvaluator.ContainsKeyAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch)))
+                    {
+                        // issue exists so just update the issue
+                        var issueNumber = await gitHubIssueEvaluator.TryGetValueAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch));
+                        Logger.LogInformation("Updating a gitHub issue number : " + issueNumber + " for the error : " + repositoryBranchUpdateHistory.ErrorMessage +
+                            " for the repository : " + repositoryBranchUpdateHistory.Repository);
+                        if (issueNumber.HasValue)
+                        {
+                            try
+                            {
+                                var issue = await client.Issue.Get(_options.Owner, _options.Repository, issueNumber.Value);
+                                var issueUpdate = issue.ToUpdate();
+                                issueUpdate.Body = issue.Body + Environment.NewLine 
```

Comment:
Is this concatenation going to make any sense?  The description is going to get duplicated a lot, and there isn't really an separate between them... so it'll just become a huge list of the description over and over.  Would adding a comment of "this got hit again" with a brief message work better?   Some of the data is duplicated (like repository and branch name), so repeating it a bunch of times in the description will be a bit confusing, I think.

We should probably at least add some headers or horizontal lines or something between the instances, to clear things up a bit.

Reply:
I added extra lines inbetween the issues. This is how it looks -> https://github.com/maestro-auth-test/maestro-test2/issues/9481

The reason for duplication is, when I call update issue it updates the existing issue's body. So the content will be lost, so by concatenation this will have older body value and new body value.

Reply:
Looking at the sample issue, I don't see any advantage of appending the same description over and over, it just made me look really hard to see if there was anything different between all of them.

What do you think of making it look a little more like this?
https://github.com/maestro-auth-test/maestro-test2/issues/9481#issuecomment-597193783